source/Ship.cpp

```diff
@@ -1518,8 +1518,8 @@ bool Ship::CanSendHail(const PlayerInfo &player, bool allowUntranslated) const
 	if(!gov || IsYours())
 		return false;
 
-	// Make sure this ship is able to send a hail.
-	if(IsDisabled() || !Crew() || Cloaking() >= 1. || GetPersonality().IsMute())
+	// Make sure this ship is able to send a hail message, unless it has the quiet or mute personality.
```

Comment:
I don't see why this comment was updated. The new part doesn't really make sense, either. This if statement is still only making sure that this ship can send a hail; new personalities don't change that. "Unless" makes it sound like "If these personalities are here, then don't check if the ship is able to send a hail."

Reply:
Revert it then?

Reply:
Yeah.